packages/projection-typeorm/test/jest-setup/jest-setup.ts

```diff
@@ -1,10 +1,14 @@
 import { DockerUtil } from '@cardano-sdk/util-dev';
 import { connectionConfig } from '../util';
+import path from 'path';
 
 module.exports = async () => {
+  const fixturesPath = path.join(__dirname, '..', '..', '..', 'cardano-services', 'test', 'jest-setup', 'snapshots');
+
   await DockerUtil.setupPostgresContainer(
     connectionConfig.username,
     connectionConfig.password,
-    connectionConfig.port.toString()
+    connectionConfig.port.toString(),
+    [`${fixturesPath}:/docker-entrypoint-initdb.d`]
```

Comment:
`projection-typeorm` is only testing the **writes** so I don't think we need to import data here

Reply:
4f6d8619f , 4f6d8619f and c64200743 should solve this